README.md

```diff
@@ -1 +1,156 @@
-# enumerable
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Enumerable] <a name="about-project"></a>
+
+**[Enumerable]** is a pair programming project where you sort, and organize different values.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Sort]**
+- **[Sort Max value, Sort Min value]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Install Ruby with RubyInstaller, then open your CMD and type 'irb'
+
+Now you can start using Ruby!
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Have installed Ruby
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd desire-folder
+  git clone https://github.com/Mike47ip/morse-code.git
+
+### Install
+
+No installation is needed.
+
+### Usage
+
+To run the project, execute the following command:
+
+No command existing until now.
+
+### Run tests
+
+To run tests, run the following command:
+
+No test have been created yet for this project.
+
+### Deployment
+
+You can deploy this project using:
+
+No deployment have made yet.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Michael Darkwah**
+
+- GitHub: [@Mike47ip](https://github.com/Mike47ip)
+- Twitter: [@ip_mikee](https://twitter.com/ip_mikee)
+- LinkedIn: [Michael Darkwah](https://linkedin.com/in/michael-darkwah-81
```

Comment:
- [ ] This is currently a broken link showing an error 404 page, please kindly fix it as it is unprofessional to leave broken links in your readme file